README.md

```diff
@@ -120,9 +165,32 @@ store.dispatch(setOrigin('https://example.com'));
 
 ```
 
-### `errors`
+### `error` Duck
+
+The error duck tracks whether One App is in an error state and what the HTTP status code is for the
+error.
+
+**Contents:**
+
+- [State Shape](#state-shape-1)
+- [Action creators](#action-creators-1)
+  - [`applicationError`](#applicationerror)
+  - [`clearError`](#clearerror)
+
+#### State Shape
+
+```js
+const state = new Map({
+  error: new Map({
+    code: Number, // HTTP error code
```

Comment:
So is this `null` if One App . is not in an error state?

Reply:
its undefined